packages/bloom/lib/commands/bloom/INFO.ts

```diff
@@ -1,11 +1,63 @@
-import { RedisArgument, Command } from '@redis/client/dist/lib/RESP/types';
+import { RedisArgument, Command, UnwrapReply, NullReply, NumberReply, TuplesToMapReply, Resp2Reply, SimpleStringReply } from '@redis/client/dist/lib/RESP/types';
+
+export type BfInfoReplyMap = TuplesToMapReply<[
+  [SimpleStringReply<'Capacity'>, NumberReply],
+  [SimpleStringReply<'Size'>, NumberReply],
+  [SimpleStringReply<'Number of filters'>, NumberReply],
+  [SimpleStringReply<'Number of items inserted'>, NumberReply],
+  [SimpleStringReply<'Expansion rate'>, NullReply | NumberReply] 
+]>;
+
+export interface BfInfoReply {
+  capacity: NumberReply;
+  size: NumberReply;
+  numberOfFilters: NumberReply;
+  numberOfInsertedItems: NumberReply;
+  expansionRate: NullReply | NumberReply;
+}
 
 export default {
   FIRST_KEY_INDEX: 1,
   IS_READ_ONLY: true,
   transformArguments(key: RedisArgument) {
     return ['BF.INFO', key];
   },
-  // TODO
-  transformReply: undefined as unknown as () => any
+  transformReply: {
+    2(reply: UnwrapReply<Resp2Reply<BfInfoReplyMap>>): BfInfoReply {
+      return {
+        capacity: reply[1],
+        size: reply[3],
+        numberOfFilters: reply[5],
+        numberOfInsertedItems: reply[7],
+        expansionRate: reply[9]
+      };
+    },
+    3(reply: UnwrapReply<BfInfoReplyMap>): BfInfoReply {
+      if (reply instanceof Map) {
+        return {
+          capacity: reply.get('Capacity') as NumberReply,
```

Comment:
The user asked for a `Map`, why are we converting it to `object`?

Reply:
We aren't defining it as a MapReply return object, we are defining it as an object return object, so the user never sees that it can be converted.

There are a few thoughts

1) what is the advantage of using a Map object over a plain object (Record<>)?  Does it acutally gain us anything?  the keys still can't be binary.

2) info objects are not generic, they are predefined data structures, so the concept of needing to "iterate" over them doesn't seem valid (but goes back to #1, what advantage would a map give in this situation).  type transformation makes sense when they improve the user experience, not when they make it more complicated for zero gain.